vertica-kafka-scheduler/README.md

```diff
@@ -249,9 +248,9 @@ vkconfig microbatch --add \
 After you [create a scheduler](#create-a-scheduler), launch the scheduler to begin scheduling microbatches. To launch a scheduler, execute a `docker run` command that does the following:
 - Mounts a configuration file as a volume.
 - Specifies the scheduler image name.
-- Mounts the local `vkafka-log-config-debug.xml` file in the container's `/opt/vertica/packages/kafka/config` directory. This file configures log messages to help troubleshoot scheduler issues.
-- Mounts the local `/log` directory in the container's `/opt/vertica/log` directory to write logs to help troubleshoot scheduler issues.
-- Passes the Docker `--user` command to specify the user.
+- Mounts `vkafka-log-config-debug.xml` from the current directory file in the container's `/opt/vertica/packages/kafka/config` directory. This file configures log messages to help troubleshoot scheduler issues.
+- Mounts `/log` from the current directory into the container's `/opt/vertica/log` directory so that log messages are written to the local `/log` directory.
+- Passes the Docker `--user` command to specify the UID that has write access on the `/log` directory.
```

Comment:
`log` not `/log` on both lines here.  Saying "/log from the current directory" doesn't make sense because / is the root directory.

Reply:
ahhh yeah gotcha--thanks!

Reply:
I also changed the wording slightly--let me know if you think it is better or I should revert.